jx_sqlite/query_table.py

```diff
@@ -278,7 +278,7 @@ def query(self, query):
 
             output = Data(
                 meta={"format": "table"},
-                header=column_names,
+                header=[split_field(h)[0] if "\\" in h else h for h in column_names],
```

Comment:
The `column_names` are wrong. I reverted back to `header=column_names` for the merge.